predict.py

```diff
@@ -96,14 +92,9 @@ def predict(
             "torch_dtype": torch.float16,
```

Comment:
Since we're cleaning up, lets also remove the unused code from here (pipe_line_args)

Reply:
Good catch. I also removed the `torch.device("cuda")`, since device is already at the top of the function on line 75.